tests/test_task_sequence.py

```
@@ -116,5 +117,18 @@ def test_required_param(self):
         with self.assertRaisesRegex(Exception, "'param1' is a required property"):
             task.validate()
 
+
+    def test_validate_sequence(self):
+        resolver = Resolver()
+        with TemporaryDirectory() as tempdir:
+            workspace = resolver.workspace_from_url(assets.path_to('kant_aufklaerung_1784/data/mets.xml'), dst_dir=tempdir)
+            params_path = Path(tempdir, 'params.json')
+            params_path.write_text('{"param1": true}')
+            with self.assertRaisesRegex(Exception, 'Input file group not contained in METS or produced by previous steps: FOO'):
+                validate_tasks([ProcessorTask.parse(x) for x in [
+                    'sample-processor-required-param -I IN -O OUT -p %s' % params_path,
```

oops, this should also raise an exception as `IN` isn't in the METS